data/coalition/coalition missions.txt

```diff
@@ -3167,3 +3166,444 @@ mission "Heliarchs Buy Jump Drive Mission"
+				decline
+			label "why not go to human space"
+			`	"I need to be available at all times, so I can answer their questions. Do not worry about me."`
+			choice
+				`	"Got it. Good luck publishing your book."`
+					goto "anna thanks you"
+
+event "beyond the hyperdrive is published"
+
+mission "Coalition: Struggling Author 5A"
+	landing
+	source
+		not near "Quaru" 0 100
+	destination "Inmost Blue"
+	to offer
+		has "event: beyond the hyperdrive is published"
+		not "Fangvin: Struggling Author 5B: offered"
+	on offer
+		conversation
+			`Checking your computer feed for the latest galactic news and trade prices, you get a reminder from your computer.`
+			`	You read it, and recall that you had helped an author named Anna Fangvin get her book, 'Beyond the Hyperdrive', published about half a year ago. You should return to <destination> and check on her.`
+				accept
+	on complete
+		conversation "anna invites you to book signing"
+
+mission "Coalition: Struggling Author 5B"
+	landing
+	source
+		near "Quaru" 0 100
+	destination "Inmost Blue"
+	to offer
+		has "event: beyond the hyperdrive is published"
+		not "Fangvin: Struggling Author 5A: offered"
+	on offer
+		conversation
+			`As you land on the spaceport, you check your messages. You see a message from Anna Fangvin, the author who you helped about half a year ago. In the message, she explains that her novel, 'Beyond the Hyperdrive', has been approved. You should return to <destination> and meet her.`
+				accept
+	on complete
+		conversation "anna invites you to book signing"
+
+conversation "anna invites you to book signing"
+	action
+		log "People" "Anna Fangvin" "Anna is an author of science fiction books. She never found success in human space, until she visited the Coalition and got her novel, 'Beyond the Hyperdrive', about a group of humans stealing jump drives from the Quarg, published. Upon publication, her book quickly became a bestseller in Kimek space."
+		log "Helped somebody named Anna Fangvin get her book published in Coalition space. Apparently, it's a bestseller now. The book was 'Beyond the Hyperdrive', something about humans stealing jump drives from the Quarg. Also found out she was ambidextrous."
+		"salary: Anna Fangvin" = 5000
+		payment 2000000
+		set "beyond the hyperdrive news can show now"
+	`The moment you land, Anna is all too happy to see you. She grabs your shoulders with both arms and in an excited tone explains that her book has been approved for publication.`
+	`	"Today, they will be publishing 'Beyond the Hyperdrive' and I will be signing copies! Thanks for everything you've done, <first>!"`
+	choice
+		`	"It got approved? That's awesome!"`
+	`	"I know, right? So do you want to join me during the book signing?"`
+	choice
+		`	"I've been with you through the whole process. Count me in!"`
```

Comment:
Player should have an option to decline here.

Reply:
Added an option not to attend in da32627b7